include/nuttx/power/battery_charger.h

```diff
@@ -125,7 +125,11 @@ struct battery_charger_operations_s
 
   /* Get the actual output voltage for charging */
 
-  int (*get_voltage)(struct battery_charger_dev_s *dev, int *value);
+  int (*get_voltage)(struct battery_charger_dev_s *dev, FAR int *value);
```

Comment:
In general all `struct battery_charger_operations_s` is missing `CODE` and `FAR`, but let's add it in the separate PR